### DISCUSSION TITLE
Chapter Duration.

### DISCUSSION THREAD

**Discussion by westhecoder** [PaulWoitaschek/Voice#1560] [Ideas]
Thank you for this amazing application.

Won't be a good idea to see the duration of the chapter when  I click to see the list of book chapters?

**Comment by Croliz** [comment]
I'd like to have this feature too, is this still not available?

**Comment by PaulWoitaschek** [comment]
It's now implicitly available on the right (not duration per chapter, but global start.

![Screenshot_20260420-142326.png](https://github.com/user-attachments/assets/3f372358-9448-425c-b17a-eef798681b39)

**Comment by Croliz** [comment]
Hi Paul,

thanks for the reply.

Yes I know that there is the global start, I would have liked a feature where you could maybe chose wether to see the global start or the total duration of the chapter.

anyways, thanks for your work
regards,
BML
________________________________

**Comment by poti1** [comment]
I see this is getting to be an old post without any traction 😕 

This seems like a quick win. Let's see if we can help out and make this into a merge request (MR) 💪

**Reply by poti1** [reply]
It's on: I'm making a fork of this repo 💪

**Comment by poti1** [comment]
Created: https://github.com/PaulWoitaschek/Voice/pull/3746
